recipes/nappy/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "nappy" %}
+{% set version = "1.1.4" %}
+{% set sha256 = "2bc123250e559c574c57ad8d7e4c05814d58f3dc45f70622b1be401d30ec1b20" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win, py3]
```

Comment:
Does this syntax work?
I would put either `[win or py3k]` or `[win and py3k]` depending on what you are after.